src/autopas/utils/SoA.h

```diff
@@ -129,7 +139,7 @@ class SoA {
    *
    * @return Number of particles.
    */
-  size_t getNumParticles() const { return soaStorage.template get<0>().size(); }
+  inline constexpr size_t getNumParticles() const { return soaStorage.template get<0>().size() - viewStart; }
```

Comment:
does `constexpr` really do anything here?

Reply:
removed in b6494b3d43368c28864d0a02778020c5a8878d30